lang/csharp/src/apache/main/Reflect/Readme.md

```diff
@@ -0,0 +1,198 @@
+# Namespace Avro.Reflect
```

Comment:
Nice write up!

Could you rename this file to all-caps, `README.md`, for the sake of consistency with other READMEs throughout the project? I believe this will also fix the Travis CI build failure due to this file not having a license header. The [`pom.xml` excludes all `README.md` files](https://github.com/apache/avro/blob/master/pom.xml#L353) from the license header check.